scripts/lib/libvirt/compute-config

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+import argparse
+import subprocess
+import sys
+
+import libvirt
+import libvirt_helpers
+
+
+def main():
+    conn = libvirt.open("qemu:///system")
+    if not conn:
+        print("Failed to open connection to the hypervisor")
+        sys.exit(1)
+
+    parser = argparse.ArgumentParser(description="Create Compute Node Config.")
+    parser.add_argument("cloud", type=str, help="Name of the cloud")
+    parser.add_argument("nodecounter", type=str, help="Node Counter")
+    parser.add_argument("macaddress", type=str, help="MAC Address")
+    parser.add_argument("cephvolume", type=str, help="Ceph Volume")
+    parser.add_argument("drbdvolume", type=str, help="DRBD Volume")
+    parser.add_argument("computenodememory", type=str,
+                        help="Compute Node Memory")
+    parser.add_argument("controllernodememory", type=str,
+                        help="Controller Node Memory")
+    parser.add_argument("libvirttype", type=str, help="Libvirt Type")
+    parser.add_argument("vcpus", type=str, help="VCPUs")
+    parser.add_argument("emulator", type=str, help="Emulator")
+    parser.add_argument("vdiskdir", type=str, help="Virtual Disk Directory")
+
+    args = parser.parse_args()
+
+    cloud = args.cloud
+    fin = libvirt_helpers.readfile("templates/compute-node.xml")
```

Comment:
Same problem here: there is a load of business logic which has nothing to do with the CLI code and should live in a separate function.